lib/view_model/access_control/tree.rb

```diff
@@ -109,51 +104,41 @@ def valid_edit_check(view, deserialize_context:, changes:)
     policy_instance_for(view).valid_edit_check(view, deserialize_context: deserialize_context, changes: changes)
   end
 
-  private
-
-  def policy_instance_for(view)
-    view_name = view.class.view_name
-    @view_policy_instances.fetch(view_name) { @always_policy_instance }
-  end
-
-  # Mix-in for traversal contexts to support saving precalculated
-  # child-editability/visibility for tree-based access control roots.
-  module AccessControlRootMixin
-    extend ActiveSupport::Concern
-
-    RootData = Struct.new(:visibility, :editability)
-
-    def initialize(*)
-      super
-      @access_control_root_data = nil
+  def store_descendent_editability(node, descendent_editability)
```

Comment:
`node` parameter is of type `ViewModel`, not `Node`.

Reply:
fixed